src/Repository/ProductCommentCriterionRepository.php

```diff
@@ -46,7 +46,7 @@ class ProductCommentCriterionRepository
      * @param Connection $connection
      * @param string $databasePrefix
      */
-    public function __construct(Connection $connection, $databasePrefix)
+    public function __construct($connection, $databasePrefix)
```

Comment:
any reason why you removed it?

Reply:
It is identified as an error by phpstan 0.12 via composer.json on my localhost (automatic phpstan does not think so :)
![composerjson_phpstan012_errors](https://github.com/PrestaShop/productcomments/assets/3759923/5ff387f2-3f6f-40bf-888f-cb64adefc204)